drivers/display/display_stm32_ltdc.c

```diff
@@ -260,6 +262,38 @@ static int stm32_ltdc_read(const struct device *dev, const uint16_t x,
 	return 0;
 }
 
+static int stm32_ltdc_display_blanking_off(const struct device *dev)
+{
+	const struct display_stm32_ltdc_config *config = dev->config;
+	int r = 0;
+
+	LOG_DBG("Turning display blanking off");
+	/* Needed for display to function correctly on boards that use the display's controller
+	 * for configuration and LTDC for data.
+	 */
+	if (device_is_ready(config->display_controller)) {
+		r = display_blanking_off(config->display_controller);
+	}
+
+	return r;
+}
+
+static int stm32_ltdc_display_blanking_on(const struct device *dev)
+{
+	const struct display_stm32_ltdc_config *config = dev->config;
+	int r = 0;
+
+	LOG_DBG("Turning display blanking on");
+	/* Needed for display to function correctly on boards that use the display's controller
+	 * for configuration and LTDC for data.
+	 */
+	if (device_is_ready(config->display_controller)) {
+		r = display_blanking_on(config->display_controller);
+	}
+
+	return r;
```

Comment:
it's usually a good practice throughtout the code base to indent error handling and keep "normal" execution inline, plus you are not handling the null case, how about somthing like

```c
if (config->display_controller == NULL) {
        return 0;
}

if (!device_is_ready(config->display_controller)) {
        LOG_ERR("Display device %s not ready", config->display_controller->name);
        return -ENODEV;
}

return display_blanking_on(config->display_controller);
```

Reply:
I think device_is_ready() handles the NULL case but can add it if it makes the code clearer.

Reply:
`device_is_ready` should handle the NULL case, yeah- but I think the idea here is that a NULL device is a different issue from a device that failed to initialize and is not ready.